tools/hooks/restrict_levelbuilder_changes.rb

```diff
@@ -2,6 +2,8 @@
 
 REPO_DIR = File.expand_path('../../../', __FILE__)
 LEVELS_DIR = File.expand_path(REPO_DIR + '/dashboard/config/scripts', __FILE__)
+WHITELISTED_FILES = %w[dashboard/config/locales/dsls.en.yml dashboard/config/locales/scripts.en.yml].map{|f|File.join(REPO_DIR, f)}
```

Comment:
scripts.en.yml is already included: %w[dashboard/config/locales/dsls.en.yml dashboard/config/locales/scripts.en.yml]